webapp/src/components/checklist_item.tsx

```diff
@@ -98,7 +98,10 @@ interface StepDescriptionProps {
 const StepDescription = (props: StepDescriptionProps) : React.ReactElement<StepDescriptionProps> => {
     const [showTooltip, setShowTooltip] = useState(false);
     const target = useRef(null);
-
+    const popoverRef = useRef(null);
+    useClickOutsideRef(popoverRef, () => {
+        setTimeout(() => setShowTooltip(false), 10);
```

Comment:
Why do we need the timeout here? Can we remove it?

Reply:
Like I mentioned above, on `useClickOutsideRef` it is working as expected, but then it is triggering the `onClick` on the `InfoIcon`.

Which is why I put the `useClickOutsideRef`  as a callback for `setTimeout` so that it trigger the `onClick` prior to `useClickOutsideRef`.

I don't think it'll be a problem, but I am open to learn about the alternative if you'd want to get into this.

Reply:
Also, I made this change as I felt it wasn't a good user experience. I'd expect to click on the same button/icon to toggle.
I can remove it tho if you want 👍

Reply:
I'm thinking now that another solution for this would be to change the current `useClickOutsideRef` hook to `useClickOutsideRefs`, and pass a list of refs instead of a single one. In the code of the hook, check that none of the refs is being clicked on. In the caller code, use it simply with `useClickOutsideRef([popoverRef, target], setShowTooltip(false))`.
I have not tested it, but it may work. Are you open to try it?

Reply:
I don't see any harm in trying. Let me raise another PR in sometime :)

Reply:
I agree -- I don't think we need it. You can probably do:
```suggestion
        setShowTooltip(false);
```

Reply:
So, do we want to try `useClickOutsideRefs` which @agarciamontoro  suggested? Or should I just remote `setTimeout` and update the PR?